ops-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+services:
+    app:
+        build:
+            context: ../../
+            dockerfile: ./ops-tools/docker/images/app/Dockerfile
+        command: sh -c "mvn package && java -jar target/docker-practice-java-0.0.1-SNAPSHOT.jar"
+        volumes:
+            - ../../:/app
+        ports:
+            - "8080:8080"
+            - "8888:8080"
+        networks:
+            - docker-practice-net
+        depends_on:
+            - redis
+            - postgresql
+
+    redis:
+        image: redis:6.0.9-alpine
+        restart: unless-stopped
+        ports:
+            - "6379:6379"
+        networks:
+            - docker-practice-net
+
+    postgresql:
+        image: postgres:14
```

Comment:
Let's use alpine images - they are lightweight and should meet our needs.

Reply:
postgres and mvn images now is alpine-versions;